CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- Filters - Improve filters genericity
```

Comment:
- Technical - Upgrade Ruby version to 2.3.4 for the CI.

Reply:
I would also add a 🛡icon to the first change:
"- Filters - Improve filters genericity. 🛡"
to show that tests have been written for this change 💪🏼